src/baseTrie.js

```diff
@@ -375,7 +377,8 @@ module.exports = class Trie {
       return onDone()
     }
 
-    this._lookupNode(root, node => {
+    this._lookupNode(root, (e, node) => {
+      if (e) { return onDone(e, node) }
```

Comment:
Can you do this with three-line code style to be consistent e.g. with the code directly below (sorry for the nitpicking 😄)?